packages/aws-cdk/test/cdk-toolkit.test.ts

```diff
@@ -241,7 +241,6 @@ describe('readCurrentTemplate', () => {
     // THEN
     expect(flatten(stderrMock.mock.calls)).toEqual(expect.arrayContaining([
       expect.stringMatching(/Could not assume bloop-lookup:here:123456789012/),
-      expect.stringMatching(/please upgrade to bootstrap version >= 5/),
```

Comment:
This is weird. Without successfully assuming the lookup role, how will we know what bootstrap version is in SSM. Maybe cached? Taking a look

But, if we are able to assume the role, we do throw in such scenario instead of a warning: https://github.com/aws/aws-cdk/blob/vkukreja/doc-update/packages/aws-cdk/lib/api/deployments.ts#L546-L549

Reply:
So it was throwing the expected error but since we catch it and do nothing, it will only show up if we have `--debug` flag enabled. I believe it needs to be shown and hence adding it with `error`